libs/execution_context/ebpf_core.c

```diff
@@ -821,7 +988,7 @@ ebpf_core_get_protocol_handler_properties(
     *minimum_request_size = 0;
     *minimum_reply_size = 0;
 
-    if (operation_id > EBPF_OPERATION_GET_PROGRAM_INFORMATION || operation_id < EBPF_OPERATION_RESOLVE_HELPER)
+    if (operation_id >= EBPF_OPERATION_MAX || operation_id < EBPF_OPERATION_RESOLVE_HELPER)
```

Comment:
EBPF_OPERATION_MAX as the size of an array is bad programming practice and we should not have such a define.  I suspect you want
```suggestion
    if (operation_id >= _countof(_ebpf_protocol_handlers) || operation_id < EBPF_OPERATION_RESOLVE_HELPER)
```